src/services/config.service.ts

```diff
@@ -75,9 +75,18 @@ export default class ConfigService {
     return this.apiService.getAdSchedule(adScheduleId);
   };
 
-  loadConfig = async (configLocation: string) => {
+  loadConfig = async (configLocation: string | undefined) => {
+    const i18n = getI18n();
+    const bundle = i18n.getResourceBundle(i18n.language, 'error');
```

Comment:
Does `title: i18n.t('error:config_invalid'),` not work?

Reply:
Thanks!